pkg/cmd/release-payload-controller/release_creation_status_controller.go

```
@@ -259,3 +271,46 @@ func computeReleaseCreationJobMessage(job *batchv1.Job) string {
+	for _, pod := range pods.Items {
+		if s := findContainerStatus(pod.Status.InitContainerStatuses, "build"); s != nil {
+			statuses = append(statuses, s)
+		} else if s := findContainerStatus(pod.Status.ContainerStatuses, "build"); s != nil {
+			statuses = append(statuses, s)
+		}
+	}
+
+	sort.Slice(statuses, func(i, j int) bool {
+		a, b := statuses[i], statuses[j]
+		if a.State.Terminated != nil && b.State.Terminated != nil {
+			return a.State.Terminated.FinishedAt.Time.After(b.State.Terminated.FinishedAt.Time)
```

golangci-lint (2.12.2)

[error] QF1008: could remove embedded field "Time" from selector

(staticcheck)